src/components/include/cos_kernel_api.h

```diff
@@ -30,7 +30,7 @@ struct cos_meminfo {
 /* Component captbl/pgtbl allocation information */
 struct cos_compinfo {
 	/* capabilities to higher-order capability tables (or -1) */
-	capid_t pgtbl_cap, captbl_cap, comp_cap;
+	capid_t pgtbl_cap, captbl_cap, comp_cap, local_pgtbl_cap;
```

Comment:
@interwq: this is to support persistent memory that is shared between non-coherent nodes.  We're porting the kernel's quiescence mechanisms to handle cache non-coherence.

@ryuxin: we do want to merge it, but we want it to likely be unused in the common configuration.  I don't think that the vkernel work wants it, and we don't want to waste the resources on the pgtbl if we don't need to.

I'd imagine we'll have the slot reserved in the captbl for captbl/pgtbl.  We'll have a specific `#define` in `cos_config.h` to activate the kernel parts that build the structures, and you'll write a specific booter that uses the global stuff.